File: C/natural.c
#include<stdio.h>
int main(void) {
  int Number, Counter = 0;
  printf("How many natural numbers do you want to print?");
  scanf("%d", &Number);
  printf("The first %d natural numbers are", Number);
  for(Counter = 1; Counter <= Number; Counter++) {
    printf(" %d", Counter);
  }
  printf(".");
  return 0;
}

File: Swift/natural.swift
print ("How many natural numbers do you want to print", terminator : "?")
var Number = Int(readLine(strippingNewline: true)!)!
var Counter = 1
print ("The first \(Number) natural numbers are", terminator : "")
while Counter <= Number {
  print (" \(Counter)", terminator : "")
  Counter = Counter + 1
}
print (".")